zipkin/src/test/java/zipkin/storage/SpanStoreTest.java

```diff
@@ -816,17 +817,16 @@ public void clientTimestampAndDurationWinInSharedSpan() {
   @Test
   public void traceWithManySpans() {
     Span[] trace = new Span[101];
-    trace[0] = TestObjects.TRACE.get(0);
+    trace[0] = Span.builder().traceId(0xf66529c8cc356aa0L).id(0x93288b4644570496L).name("get")
+      .timestamp(TODAY * 1000).duration(350 * 1000L)
+      .addAnnotation(Annotation.create(TODAY * 1000, SERVER_RECV, WEB_ENDPOINT))
+      .addAnnotation(Annotation.create((TODAY + 350) * 1000, SERVER_SEND, WEB_ENDPOINT))
+      .build();
 
     IntStream.range(0, 100).forEach(i -> {
-      Span s = TestObjects.TRACE.get(1);
-      trace[i + 1] = s.toBuilder()
-          .id(s.id + i)
-          .timestamp(s.timestamp + i)
-          .annotations(s.annotations.stream()
-              .map(a -> Annotation.create(a.timestamp + i, a.value, a.endpoint))
-              .collect(toList()))
-          .build();
+      trace[i + 1] = Span.builder().traceId(trace[0].traceId).parentId(trace[0].id).id(i).name("")
```

Comment:
this test is also testing "raw spans". To make sure this passes, we should make a span which has no authoritative timestamp (client spans do have an authoritative timestamp and that was the prior test data)